torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1362,6 +1362,40 @@ def _make_tensor_helper(shape, low=None, high=None):
 
     return samples
 
+def sample_inputs_unfold(op_info, device, dtype, requires_grad):
+    test_cases = (
```

Comment:
How long do these test cases take to run?

Sample inputs are used in grad and gradgrad tests, which are time consuming, especially for complex inputs. Once the CI is working let's pull these test timings and see if we should reduce these sizes.